foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/model2/ChatCompletionFunctionCall.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2025 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+
+/*
+ * Azure OpenAI Service API
+ * Azure OpenAI APIs for completions and search
+ *
+ * The version of the OpenAPI document: 2024-10-21
+ *
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package com.sap.ai.sdk.foundationmodels.openai.model2;
```

Comment:
Have you found a better name than model2 yet?

Reply:
Thanks for asking!

Some suggestions:
* [ ] generated
* [ ] dto
* [ ] objects
* [ ] data
* [ ] domain

Reply:
I like `generated` or `generated.model`